packages/next/src/build/webpack/plugins/next-types-plugin.ts

```diff
@@ -267,10 +269,10 @@ declare namespace __next_route_internal_types__ {
   type OptionalCatchAllSlug<S extends string> =
     S extends \`\${string}\${SearchOrHash}\` ? never : S
 
-  type StaticRoutes = ${staticRouteTypes}
-  type DynamicRoutes<T extends string = string> = ${dynamicRouteTypes}
+  type StaticRoutes = ${staticRouteTypesFallback}${staticRouteTypes}
+  type DynamicRoutes<T extends string = string> = ${dynamicRouteTypesFallback}${dynamicRouteTypes}
```

Comment:
```suggestion
  type StaticRoutes = ${staticRouteTypes || 'string'}
  type DynamicRoutes<T extends string = string> = ${dynamicRouteTypes || 'string'}
```

For simplicity maybe?

Reply:
Done :)